elements/pfe-navigation/src/sass-includes/_variables-mixins.scss

```diff
@@ -0,0 +1,507 @@
+// Please see the pfe-sass README for guidance on using these tools
+@import "../../pfe-sass/pfe-sass";
+
+$LOCAL: navigation;
+
+/// PFE-Local setup variables
+/// These variables will be namespaced and become css vars
+$LOCAL-VARIABLES: (
+  gutter: 32px,
+  accordion-transition: height 0.25s ease-in-out,
+  content-max-width: 1136px,
+  FontFamily: #{"Red Hat Text", "RedHatText", Arial, Helvetica, sans-serif},
+  ZIndex: pfe-zindex(navigation),
+  overlay--Background: rgba(21, 21, 21, 0.5), // @todo Deviates from PFE overlay var?
+
+  nav-bar: (
+    Height: 72px,
+    Background: #151515, // @todo Doesn't exist in our palette?
+    Color--default: pfe-var(ui-base--text),
+    Color--active: #151515,
+    toggle--BackgroundColor--active: pfe-var(surface--lightest),
+    highlight-color: #e00, // Used for hover over menu items and alerts
+    Color--on-highlight: pfe-var(text--on-saturated),
+  ),
+
+  dropdown: (
+    Color: #151515,
+    Color--on-highlight: #fff,
+    Color--secondary: #6A6E73,
+    Background: pfe-var(surface--lightest),
+    link--Color: #06c, // @todo Is a different color than pfe-var(link)..
+    link--Color--hover: #036, // @todo Doesn't exist in our palette?
+    headings--Color: #464646,
+    headings--single-column--Color: #6A6E73, // @todo Do we need two different heading colors?
+    separator--Border: 1px solid pfe-var(surface--border),
+    BoxShadow: 0 1px 2px rgba(0, 0, 0, 0.12),
+    arrow--color: #B8BBBE, // @todo look at chevron mixin
+    arrow--color--active: #6A6E73,
+    highlight-color: #e00, // Used for hover over menu items and alerts
```

Comment:
I know this isn't red, but I'd suggest using `pfe-var(ui-accent)`  and `pfe-var(ui-accent--focus)` and then setting the value of that var to red in your theme file, so that the PFE project stays closer to the PF core design palette.

```
  --pfe-theme--color--ui-accent: #ee0000;
```

Reply:
🤔 That could be a bit of a pain for a general roll out, each site would have to set that var in a separate CSS file. I'll update it, but want to keep an eye on how we can make that easy.

Reply:
The internal branding theme assets are hosted on static and here is the documentation on The Source documenting and recommending it's use: 

https://source.redhat.com/departments/marketing/digitalmarketingstrategy/digital_business_wiki/tools_for_creating_a_red_hat_branded_web_page_or_application#red-hat-theme (internal-only URL)

Reply:
https://static.redhat.com/libs/redhat/redhat-theme/latest/advanced-theme.css (internal)